substrate/frame/revive/src/lib.rs

```diff
@@ -2056,6 +2177,19 @@ sp_api::decl_runtime_apis! {
 macro_rules! impl_runtime_apis_plus_revive {
 	($Runtime: ty, $Executive: ty, $EthExtra: ty, $($rest:tt)*) => {
 
+		impl $crate::evm::runtime::SetWeightLimit for RuntimeCall {
```

Comment:
ah that's where you implement it, nice, I guess  we should rename the macro it's not just about the runtime_api anymore